iDiff/utils/docker_utils.go

```diff
@@ -110,6 +110,33 @@ type HistDiff struct {
 	Dels   []string
 }
 
+// getImageHistory shells out the docker history command and returns a list of history response items.
+// The history response items contain only the Created By information for each event.
+func getImageHistory(image string) ([]img.HistoryResponseItem, error) {
+	imageID := image
+	var err error
+	var history []img.HistoryResponseItem
+	histArgs := []string{"history", "--no-trunc", imageID}
+	dockerHistCmd := exec.Command("docker", histArgs...)
+	var response bytes.Buffer
+	dockerHistCmd.Stdout = &response
+	if err := dockerHistCmd.Run(); err != nil {
+		if exiterr, ok := err.(*exec.ExitError); ok {
+			if status, ok := exiterr.Sys().(syscall.WaitStatus); ok && status.ExitStatus() > 0 {
```

Comment:
don't need to check if `status.ExitStatus() > 0` here (if you're in this line we already know the command has exited with a non-zero exit code, and this will also ignore negative error codes)